Model/Api/Debug.php

```diff
@@ -79,14 +111,187 @@ public function debug()
 		# populate platform version
 		$result->setPlatformVersion($this->productMetadata->getVersion());
 
-		$boltSettings = [];
-		$boltSettings[] = $this->boltConfigSettingFactory->create();
-		$result->setBoltConfigSettings($boltSettings);
+		# populate bolt config settings
+		$result->setBoltConfigSettings($this->getBoltSettings());
 
 		$otherPluginVersions = [];
 		$otherPluginVersions[] = $this->pluginVersionFactory->create();
 		$result->setOtherPluginVersions($otherPluginVersions);
 
 		return $result;
 	}
+
+	/**
+	 * Get all bolt configuration settings
+	 *
+	 * @return BoltConfigSetting[]
+	 */
+	private function getBoltSettings()
+	{
+		$boltSettings = [];
+
+		// Active
+		$boltSettings[] = $this->boltConfigSettingFactory->create()
+		                                                 ->setName('active')
+		                                                 ->setValue(var_export($this->configHelper->isActive(), true));
+		// Title
+		$boltSettings[] = $this->boltConfigSettingFactory->create()
+		                                                 ->setName('title')
+		                                                 ->setValue($this->configHelper->getTitle());
+		// API Key (obscured)
+		$boltSettings[] = $this->boltConfigSettingFactory->create()
+		                                                 ->setName('api_key')
+		                                                 ->setValue(SecretObscurer::obscure($this->configHelper->getApiKey()));
+		// Signing Secret (obscured)
+		$boltSettings[] = $this->boltConfigSettingFactory->create()
+		                                                 ->setName('signing_secret')
+		                                                 ->setValue(SecretObscurer::obscure($this->configHelper->getSigningSecret()));
+		// Publishable Key for Checkout (obscured)
+		$boltSettings[] = $this->boltConfigSettingFactory->create()
```

Comment:
Publishable keys are public - no need to obscure them.

Reply:
fixed